lib/Board/board.js

```diff
@@ -61,6 +61,17 @@ export class Board {
             : this.cells = this.cells;
     };
 
+    // public isFilledWithTokens(): Boolean
+    isFilledWithTokens() {
+        let hasEmptyCells = false;
+        const isEmptyCell = (row, col) => row[col] === ('');
+        this.cells.forEach((row, col) => { isEmptyCell(row, col)
+            ? hasEmptyCells = true
+            : '';
+        });
+        return !hasEmptyCells;
```

Comment:
Perhaps negating [every](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) is a more appropriate function than `forEach` 🤔

Reply:
This suggestion was much appreciated, Naomi! I was able to condense the code even further by using the every() method here, and instead of negating the value of the variable, I was able to immediately return the results of the every() fn which directly calls for isEmptyCell()'s negated boolean value return, instead.